default.nix

```
@@ -6,73 +6,124 @@ with pkgs;
 
 let
   root = ./.;
-  mainnet = {
+  protocol005 = rec {
+    protocolName = "005_PsBabyM1";
+    binarySuffix = builtins.replaceStrings ["_"] ["-"] protocolName;
+  };
+  mainnet = rec {
     rev = "94f779a7";
     sha256 = "16lxilng5q8fr2ll6h4hf7wlvac6nmw4cx10cbgzj5ks090bl97r";
     patchFile = ./nix/fix-mainnet.patch;
+    protocol = protocol005;
   };
-  babylonnet = {
+  babylonnet = rec {
     rev = "b8731913";
     sha256 = "1pakf1s6bg76fq42mb8fj1immz9g9wwimd522cpx8k28zf0hkl5i";
     patchFile = ./nix/fix-babylonnet.patch;
+    protocol = protocol005;
   };
-  tezos-client-static-mainnet = import ./nix/static.nix mainnet;
-  tezos-client-static-babylonnet = import ./nix/static.nix babylonnet;
-  binary-mainnet = "${tezos-client-static-mainnet}/bin/tezos-client";
-  binary-babylonnet = "${tezos-client-static-babylonnet}/bin/tezos-client";
-  packageDesc-mainnet = {
-    project = "tezos-client-mainnet";
-    version = toString timestamp;
-    bin = binary-mainnet;
-    arch = "amd64";
-    license = "MIT";
-    dependencies = "";
-    maintainer = "Serokell https://serokell.io";
-    licenseFile = "${tezos-client-static-mainnet}/LICENSE";
-    description = "CLI client for interacting with tezos blockchain";
-    gitRevision = mainnet.rev;
-    branchName = "mainnet";
-  };
+  tezos-static-mainnet = import ./nix/static.nix mainnet;
```

not sure if nix does common subexpr elimination like this, please move to separate `let` entry?
```suggestion
  static-nix = import ./nix/static.nix;
  tezos-static-mainnet = static-nix mainnet;
```